File: 31-01-2024/Questions/Cpp Solutions/Solution3.cpp
#include <stdio.h>

int main()
{
	char coffee[3][30] = {"Espresso Coffee", "Cappuccino Coffee", "Latte Coffee"};
	char tea[8][30] = {"Plain Tea", "Assam Tea", "Ginger Tea", "Cardamom Tea", "Masala Tea", "Lemon Tea", "Green Tea", "Organic Darjeeling Tea"};
	char soup[4][30] = {"Hot and Sour Soup", "Veg Corn Soup", "Tomato Soup", "Spicy Tomato Soup"};
	char drink[3][30] = {"Hot Chocolate Drink", "Badam Drink", "Badam-Pista Drink"};
	char ch;
	int item;

	scanf("%c\n%d", &ch, &item);
	if (ch != 'c' && ch != 'C' && ch != 't' && ch != 'T' && ch != 's' && ch != 'S' && ch != 'd' && ch != 'D')
	{
		printf("INVALID OPTION");
	}
	else if (ch == 'c' || ch == 'C')
	{
		if (item < 0 || item > 3)
		{
			printf("INVALID OPTION");
		}
		else
		{
			printf("Welcome to CCD!\n");
			printf("Enjoy Your %s!", coffee[item - 1]);
		}
	}
	else if (ch == 't' || ch == 'T')
	{
		if (item < 0 || item > 8)
		{
			printf("INVALID OPTION");
		}
		else
		{
			printf("Welcome to CCD!\n");
			printf("Enjoy Your %s!", tea[item - 1]);
		}
	}
	else if (ch == 's' || ch == 'S')
	{
		if (item < 0 || item > 4)
		{
			printf("INVALID OPTION");
		}
		else
		{
			printf("Welcome to CCD!\n");
			printf("Enjoy Your %s!", soup[item - 1]);
		}
	}
	else if (ch == 'd' || ch == 'D')
	{
		if (item < 0 || item > 4)
		{
			printf("INVALID OPTION");
		}
		else
		{
			printf("Welcome to CCD!\n");
			printf("Enjoy Your %s!", drink[item - 1]);
		}
	}
}

File: 31-01-2024/Questions/Python Solution/Solution3.py
coffee=['Espresso Coffee','Cappuccino Coffee','Latte Coffee']

tea=['Plain Tea','Assam Tea','Ginger Tea','Cardamom Tea','Masala Tea','Lemon Tea','Green Tea','Organic Darjeeling Tea']

soups=['Hot and Sour Soup','Veg Corn Soup','Tomato Soup','Spicy Tomato Soup']

beverages=['Hot Chocolate Drink','Badam Drink','Badam-Pista Drink']

ch=input() #c

item=int(input()) #1

if ch!='c' and ch!='C' and ch!='t' and ch!='T' and ch!='s' and ch!='S' and ch!='b' and ch!='B':
    print('INVALID OPTION')

elif ch=='c' or ch=='C':
    if item<=0 or item>3:
        print('INVALID OPTION')

    else:
        print('Welcome to CCD!')
        print('Enjoy Your ' + str(coffee[item-1])+'!')

elif ch=='t' or ch=='T':
    if item<=0 or item>8:
        print('INVALID OPTION')

    else:
        print('Welcome to CCD!')
        print('Enjoy Your ' + str(tea[item-1])+'!')

elif ch=='s' or ch=='S':
    if item<=0 or item>4:
        print('INVALID OPTION')

    else:
        print('Welcome to CCD!')
        print('Enjoy Your ' + str(soups[item-1])+'!')

elif ch=='b' or ch=='B':
    if item<=0 or item>3:
        print('INVALID OPTION')

    else:
        print('Welcome to CCD!')
        print('Enjoy Your ' + str(beverages[item-1])+'!')
